clients/js/networks.ts

```diff
@@ -98,6 +100,14 @@ const MAINNET = {
     rpc: undefined,
     key: undefined,
   },
+  pythnet: {
+    rpc: "http://api.pythnet.pyth.network:8899/",
+    key: get_env_var("SOLANA_KEY"),
```

Comment:
This should be a different key should it not? Perhaps `PYTHNET_KEY` or is it set to this to work out of the box in tilt?

Reply:
Pythnet uses the same wallet as Solana (like all the EVM changes use the ETH_KEY). I did change both Solana and pythnet to use SOLANA_KEY_TESTNET for testnet though.

Reply:
definitely a separate key

Reply:
on that note, the `SOLANA_KEY` in testnet should also be `SOLANA_KEY_TESTNET`